apps/pom.xml

```diff
@@ -318,6 +320,18 @@
 				<version>${junit.version}</version>
 				<scope>test</scope>
 			</dependency>
+
+			<dependency>
+				<groupId>com.newrelic.telemetry</groupId>
+				<artifactId>dropwizard-metrics-newrelic</artifactId>
+				<version>${newrelic.version}</version>
+			</dependency>
+
+			<dependency>
+				<groupId>com.newrelic.telemetry</groupId>
+				<artifactId>telemetry-http-okhttp</artifactId>
+				<version>${newrelicHTTP.version}</version>
+			</dependency>
 		</dependencies>
 	</dependencyManagement>
 
```

Comment:
reminder comment to run the perf regression and mention the results in the PR description

Reply:
reminder comment to run snyk locally and mention the results in the PR description

Reply:
Left comments in the PR with the results for both

Reply:
thank you